### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+* **New Data Source:** `google_monitoring_istio_canonical_service` ([#8789](https://github.com/hashicorp/terraform-provider-google/pull/8789))
+* **New Resource:** `google_apigee_instance_attachment` ([#8795](https://github.com/hashicorp/terraform-provider-google/pull/8795))
+* **New Resource:** `google_gke_hub_membership` ([#8755](https://github.com/hashicorp/terraform-provider-google/pull/8755))
+* **New Resource:** `google_tags_tag_value` ([#8800](https://github.com/hashicorp/terraform-provider-google/pull/8800))
+
+IMPROVEMENTS:
+* added support for Apple silicon chip (updated to go 1.16) ([#3057](https://github.com/hashicorp/terraform-provider-google-beta/pull/3057))
+* container: 
+  * added support for GKE Autopilot in `google_container_cluster`([#3101](https://github.com/hashicorp/terraform-provider-google-beta/pull/3101))
+  * added `enable_l4_ilb_subsetting` (beta) and `private_ipv6_google_access` fields to `google_container_cluster` ([#3095](https://github.com/hashicorp/terraform-provider-google-beta/pull/3095))
```
Comment (c2thorn) [inline CHANGELOG.md:13]: These are google-beta links + the formatting is a bit off

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+* sql: changed the default timeout of `google_sql_database_instance` to 30m from 20m ([#8802](https://github.com/hashicorp/terraform-provider-google/pull/8802))
+
+BUG FIXES:
+* bigquery: fixed issue where you couldn't extend an existing `schema` with additional columns in `google_bigquery_table` ([#8803](https://github.com/hashicorp/terraform-provider-google/pull/8803))
+* cloudidentity: `google_cloud_identity_groups` and `google_cloud_identity_group_memberships ` will respect the `user_project_override` and `billing_project` configurations and send the appropriate headers to establish a quota project ([#8762](https://github.com/hashicorp/terraform-provider-google/pull/8762))
```
Comment (c2thorn) [inline CHANGELOG.md:18]:
```suggestion
* cloudidentity: modified `google_cloud_identity_groups` and `google_cloud_identity_group_memberships ` to respect the `user_project_override` and `billing_project` configurations and send the appropriate headers to establish a quota project ([#8762](https://github.com/hashicorp/terraform-provider-google/pull/8762))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+* compute: added minimum for `scopes` field to `google_compute_instance` resource ([#8801](https://github.com/hashicorp/terraform-provider-google/pull/8801))
+* notebooks: fixed permadiff on labels for `google_notebook_instance` ([#8799](https://github.com/hashicorp/terraform-provider-google/pull/8799))
+* secretmanager:  set required on `secrest_data` in `google_secret_manager_secret_version` ([#8797](https://github.com/hashicorp/terraform-provider-google/pull/8797))
```
Comment (c2thorn) [inline CHANGELOG.md:21]:
```suggestion
* secretmanager: set required on `secrest_data` in `google_secret_manager_secret_version` ([#8797](https://github.com/hashicorp/terraform-provider-google/pull/8797))
```

File: CHANGELOG.md (hunk 1 above)
Comment (c2thorn) [reply CHANGELOG.md:13]: Oh I see, it's all a block for container

Comment (ScottSuarez) [reply CHANGELOG.md:13]: yesssssir

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
 ## 3.63.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_monitoring_istio_canonical_service` ([#8789](https://github.com/hashicorp/terraform-provider-google/pull/8789))
+* **New Resource:** `google_apigee_instance_attachment` ([#8795](https://github.com/hashicorp/terraform-provider-google/pull/8795))
+* **New Resource:** `google_gke_hub_membership` ([#8755](https://github.com/hashicorp/terraform-provider-google/pull/8755))
+* **New Resource:** `google_tags_tag_value` ([#8800](https://github.com/hashicorp/terraform-provider-google/pull/8800))
+
+IMPROVEMENTS:
+* added support for Apple silicon chip (updated to go 1.16) ([#3057](https://github.com/hashicorp/terraform-provider-google-beta/pull/3057))
```
Comment (rileykarson) [inline CHANGELOG.md:10]: This should link this tpg PR

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,26 @@
 ## 3.63.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_monitoring_istio_canonical_service` ([#8789](https://github.com/hashicorp/terraform-provider-google/pull/8789))
+* **New Resource:** `google_apigee_instance_attachment` ([#8795](https://github.com/hashicorp/terraform-provider-google/pull/8795))
+* **New Resource:** `google_gke_hub_membership` ([#8755](https://github.com/hashicorp/terraform-provider-google/pull/8755))
+* **New Resource:** `google_tags_tag_value` ([#8800](https://github.com/hashicorp/terraform-provider-google/pull/8800))
```
Comment (rileykarson) [inline CHANGELOG.md:7]: This is beta-only

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,26 @@
 ## 3.63.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_monitoring_istio_canonical_service` ([#8789](https://github.com/hashicorp/terraform-provider-google/pull/8789))
+* **New Resource:** `google_apigee_instance_attachment` ([#8795](https://github.com/hashicorp/terraform-provider-google/pull/8795))
+* **New Resource:** `google_gke_hub_membership` ([#8755](https://github.com/hashicorp/terraform-provider-google/pull/8755))
```
Comment (rileykarson) [inline CHANGELOG.md:6]: This is beta-only

File: CHANGELOG.md (hunk 1 above)
Comment (rileykarson) [inline CHANGELOG.md:13]: The beta part should be omitted from the GA changelog

**Pull request merged**